recipes/razers3/meta.yaml

```diff
@@ -1,25 +1,17 @@
+{% set version = "3.5.8" %}
+
 package:
   name: razers3
-  version: '3.5.3'
+  version: {{ version }}
 
 source:
-  url: http://packages.seqan.de/seqan-src/seqan-src-2.1.1.tar.gz
-  sha256: df083d5939d7e73fd00bbc6bb291ac7ea7b8d446cf89a9758fdffdc0eca9ec1c
+  - url:  http://packages.seqan.de/razers3/razers3-{{ version }}-Mac-x86_64.zip                 # [osx]
```

Comment:
@FelixMoelder we do prefer to compile from source, it that possible?

Reply:
I tried that before.
But as the VM comes with Xcode 9 it tries to build for older macOS versions (below macOS 12) which are not supported.

Reply:
What works is to build the linux binary from source as before and use the precompiled binary for macOS.

Reply:
Xref: https://github.com/bioconda/bioconda-recipes/pull/21180